sys/parsoid.js

```diff
@@ -621,9 +621,7 @@ PSP.transformRevision = function(hyper, req, from, to) {
             original: original
         };
         if (from === 'sections') {
-            body2.html = {
-                body: replaceSections(original, parseSections(req))
-            };
+            body2.html = replaceSections(original, parseSections(req));
```

Comment:
This would remove the per-element header / body framing, which I don't think we ever discussed. Unless there are strong reasons for dropping the generality & extensibility of the mime-like structure, I would recommend to not change the pagebundle format in incompatible ways.